configure.ac

```diff
@@ -2122,6 +2117,24 @@
 
     AM_CONDITIONAL([HAVE_LUA], [test "x$enable_lua" != "xno"])
 
+    # If Lua is enabled, test the integer size.
```

Comment:
This looks like a version check, not a size check. Wouldn't is make sense to do something like
```c
#if sizeof(Lua_Integer) == 8
#error
#endif